miner/worker.go

```diff
@@ -991,10 +985,30 @@ func (w *worker) commit(uncles []*types.Header, interval func(), update bool, st
 	return nil
 }
 
+// deep copy current receipts. This is should be used whene an asynchronus process can modify
```

Comment:
The method doc does not conform to go style, please start with the method name, and expand from there.

Reply:
OK. I didn't use the prefix because it's a private method.